grpc-api/src/test/app.d/01-groovy-b.groovy

```diff
@@ -0,0 +1,4 @@
+import io.deephaven.appmode.ApplicationContext
+
+app = ApplicationContext.get()
```

Comment:
We should be careful in examples. `app` escapes to the global query scope. Likely, want to do `def app = ...`